x-pack/plugins/beats/index.js

```diff
@@ -4,13 +4,22 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { registerApiRoutes } from './server/routes/api';
 import { PLUGIN } from './common/constants';
 
+const HOURS_IN_SECONDS = 60 * 60;
```

Comment:
I'm assuming we're keeping this here just because of how generic it is? I.E. we don't need to track a dedicated constant value in a common folder for something unrelated to the eccentricities of the product.

Reply:
Yeah, how about we leave it here for now and lift it up higher when we need a similar conversion elsewhere?

Reply:
Fine by me!